tests/utilities/test_memory.py

```diff
@@ -28,3 +30,24 @@ def test_recursive_detach():
     assert y["foo"].device.type == "cpu"
     assert y["bar"]["baz"].device.type == "cpu"
     assert not y["bar"]["baz"].requires_grad
+
+
+def test_get_model_size_mb():
+    model = BoringModel()
+
+    size_bytes = get_model_size_mb(model)
+
+    # Size will be python version dependent.
+    assert size_bytes == 0.001319 or size_bytes == 0.001409
+
+
+def test_get_sparse_model_size_mb():
+    class BoringSparseModel(BoringModel):
+        def __init__(self):
+            super().__init__()
+            self.layer = nn.Parameter(torch.ones(32).to_sparse())
+
+    model = BoringSparseModel()
+    size_bytes = get_model_size_mb(model)
+
+    assert size_bytes == 0.001511 or size_bytes == 0.001665
```

Comment:
you can use `math.isclose()` for the floating point comparisons in case of flakiness